app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe) -> str:
```

Comment:
if you use type annotation add it for cafe also

Reply:
Thanks! Fixed